### ISSUE TITLE
Wiki nicht aktiviert

### ISSUE THREAD

**Issue by sulimma** [ewaldshof/ewhome#27]
@scy :
Ich würde gerne Dokumentation schreiben, für das Repository ist aber kein Wiki aktiviert und ich habe auch nicht die Rechte, die Einstellungen des Repositories zu bearbeiten.

Bitte ändere beides.

Kolja

**Comment by scy** [comment]
Ich bin dafür leider der falsche Ansprechpartner, habe die Rechte ebensowenig. Ich nehme an, da muss @swaldtmann ran.

**Comment by swaldtmann** [comment]
Jo, habe ich gerade gesehen und das wiki aktiviert. Wie ich @sulimma die Rechte geben kann, habe ich noch nicht gefunden ...
